ostd/src/arch/x86/iommu/mod.rs

```diff
@@ -9,7 +9,8 @@ mod invalidate;
 mod registers;
 
 pub(crate) use dma_remapping::{has_dma_remapping, map, unmap};
-pub(crate) use interrupt_remapping::{alloc_irt_entry, has_interrupt_remapping, IrtEntryHandle};
+pub(in crate::arch) use interrupt_remapping::has_interrupt_remapping;
+pub(crate) use interrupt_remapping::{alloc_irt_entry, IrtEntryHandle};
```

Comment:
`has_interrupt_remapping` has been changed to `in crate::arch`. Shouldn't we also change `alloc_irt_entry` and `IrtEntryHandle`? I `grep`'ed the code and did not find any usages outside of `crate::arch`.